src/adapters/vertoz.js

```diff
@@ -0,0 +1,89 @@
+var utils = require('../utils.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader.js');
+
+
+var VertozAdapter = function VertozAdapter() {
+  
+  const BASE_URI='//banner.vrtzads.com/vzhbidder/bid?';
+  const BIDDER_NAME='vertoz';
+  const QUERY_PARAM_KEY='q';
+
+
+  function _callBids(params) {
+
+    var bids = params.bids || [];
+
+    for (var i = 0; i < bids.length; i++) {
+
+      var bid = bids[i];
+    	 	
+      let adspaceid = utils.getValue(bid,'placementCode');
+
+      let cb = Math.round(new Date().getTime() / 1000);
+
+      let vzEndPoint = BASE_URI;
+
+      let reqParams = bid.params || {};
+
+      let placementId = utils.getValue(reqParams,'placementId');
+     	
+      if(utils.isEmptyStr(placementId)||utils.isEmptyStr(adspaceid)){
+        utils.logError('missing params:',BIDDER_NAME,'Enter valid vzPlacementId or check placementCode');
+        return;
+      }
+
+      let reqSrc = utils.getTopWindowLocation().href;
+
+      var vzReq = {
+        _vzPlacementId:placementId,
+        _rqsrc:reqSrc,
+        _cb:cb,
+        _adspaceid:adspaceid
+      };
+	
+      let queryParamValue=JSON.stringify(vzReq);
+      
+      vzEndPoint = utils.tryAppendQueryString(vzEndPoint, QUERY_PARAM_KEY, queryParamValue);
+      	
+      adloader.loadScript(vzEndPoint);
+    }
+  }
+
+  $$PREBID_GLOBAL$$.vzResponse = function (vertozResponse) {
+
+    let bidRespObj = vertozResponse;
+   
+    var bidObject;
+
+
+    if (bidRespObj.cpm) {
+   
+	
+      bidObject = bidfactory.createBid(1);
```

Comment:
Please add bid object as second parameter in createBid call. Also it would be great if you can use STATUS.GOOD constant instead of 1 defined in [constants.json](https://github.com/prebid/Prebid.js/blob/master/src/constants.json#L14) file